### DISCUSSION TITLE
Is it possible to make normal dom elements context providers?

### DISCUSSION THREAD

**Discussion by gipoezcan** [lit/lit#4690] [Q&A] [answered]
I read the context documentation and got the impression that it should be possible to respond to context events from a non-lit, standard DOM element. However, I am having trouble understanding how to do this. Could someone please provide a minimal example or point me in the right direction?

As for why this is necessary: due to legacy reasons, I cannot change any common parents of two separate lit element instances to a lit element with a provider. Therefore, I need to attach the provider to a standard DOM element.

**Comment by augustjk** [comment, marked as answer]
Yes it is possible to use the `ContextProvider` directly with a standard element but you need to call `hostConnected` must be called if the provider is being added after the consumer with a context root.

https://github.com/lit/lit/blob/main/packages/context/README.md#:~:text=ContextProvider%20can%20also%20be%20used%20with%20plain%20HTML%20elements.

**Reply by gipoezcan** [reply]
I can't believe I missed that, I'm sorry, I didn't mean to make anyone search the docs for me.

Anyway, thank you very much!
